### ISSUE TITLE
Handle compilation errors when protobuf message ends with Grpc

### ISSUE THREAD

**Issue by krmahadevan** [Adven27/grpc-wiremock#27]
**Describe the bug**

The `grpc-wiremock` tool assumes that in the generated code from the protobuf messages, only service classes would have its name end with `Grpc.java`. But a user can still end up defining a message which ends with `Grpc`. So when a message ends with `Grpc`, the tool ends up assuming that the message class is a service class and ends up adding a reference to the abstract `ImplBase` for it which does not exist.

**To Reproduce**

1. Use the below `.proto` file

<details>

```proto
syntax="proto3";
package com.rationaleemotions;

option java_multiple_files=true;

message InputGrpc {
    string name = 1;
}

message OutputGrpc {
    string message = 1;
}

service HelloWorld {
    rpc greetUser (InputGrpc) returns (OutputGrpc);
}
```

</details>

2. Add a reference to the above proto file directory in the build.gradle and then run `./gradlew clean generateJava`
3. Now from within the IDE, open up the generated Java class `src/main/java/io/adven/grpc/wiremock/Translator.java`
4. You will see the below mentioned problematic lines

```java
@Service class ComRationaleemotionsHelloWorld extends com.rationaleemotions.HelloWorldGrpc.HelloWorldImplBase {}
@Service class ComRationaleemotionsInput extends com.rationaleemotions.InputGrpc.InputImplBase {}
@Service class ComRationaleemotionsOutput extends com.rationaleemotions.OutputGrpc.OutputImplBase {}

```

In the above generated code, only `HelloWorldGrpc` is a valid service class and so `HelloWorldImplBase` exists inside it. But this is not the case with `InputGrpc` and `OutputGrpc`

**Expected behavior**

The tool should not just rely on file names to distinguish service classes from message classes but it should be able to skip such ambiguous message classes that look like a service class based on file name.

**Screenshots**

**Additional context**

**Comment by krmahadevan** [comment]
I have raised a PR for this. It would be really great if this can be accepted and then an image released into docker hub.

I stumbled into this issue at work when I was trying to use this library for doing grpc mocking. I dont want to end up maintaining a docker image at work and would like to continue using this library with the fix.

Oh btw, thank you so much for this wonderful library. Awesome work 👍
